src/plugins/azureBasePlugin.ts

```diff
@@ -5,7 +5,7 @@ import { Utils } from "../shared/utils";
 export abstract class AzureBasePlugin {
   public constructor(
     protected serverless: Serverless,
-    protected options: Serverless.Options,
+    protected options: Serverless.Options & Serverless.AzureLoginOptions,
```

Comment:
Remove ` & Serverless.AzureLoginOptions` here. `AzureBasePlugin` should have nothing specific to other plugins

Reply:
I fixed this issue after realizing that it was the reason behind failed build